### PR TITLE
Load user instead of creating fake ones when joining or hosting lobbies.

### PR DESCRIPTION
**Pull request by VictorSchneuwly**
> Linked to task #107

Load the user from the phone instead of creating fake ones when joining or hosting lobbies.

### CODE REVIEW

**Comment by VictorSchneuwly** [comment]
In order to make this work, we need to first merge the pull request of @BartLucien considering the update of users.

File: app/src/main/java/com/github/fribourgsdp/radio/User.kt [hunk 1]
```diff
@@ -149,4 +149,12 @@ data class User (var name: String, val color: Int) {
     fun getSpotifyPlaylistUId(playlistName: String): String? {
         return playlistNamesToSpotifyId[playlistName]
     }
+
```
Comment (sim0mo) [inline app/src/main/java/com/github/fribourgsdp/radio/User.kt:152]: Good idea to use inheritance !

**Review by BartLucien — APPROVED** [review]
It looks good to me, as we talked IRL, it "just" replace the the name with the ID of the user everywhere it is needed and load the user instead of creating a fake one.

**Review by Flamantrose321 — APPROVED** [review]
Everything is good for me. It is indeed much more logical to identify a user by his id which is unique whereas several users can have the same name.

**Pull request merged**